### ISSUE TITLE
Berlin building shapes

### ISSUE THREAD

**Issue by tordans** [osmlab/editor-layer-index#646]
## URL of the imagery
https://fbinter.stadt-berlin.de/fb/index.jsp?loginkey=zoomStart&mapId=wmsk_webatlasberlin@senstadt

Example image based on the URL from the PR:
https://fbinter.stadt-berlin.de/fb/wms/senstadt/wmsk_webatlasberlin?FORMAT=image/png&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&SRS=EPSG:4326&WIDTH=256&HEIGHT=256&BBOX=13.45001220703126,52.47274306920924,13.451385498046882,52.47357958606801

## Two letter country code
DE

## Desired layer name
Berlin WebAtlas (Straßen, Hausumringe & -Nr.)

## License
Same as the other Berlin Layer (Areal Images).

https://fbinter.stadt-berlin.de/fb/index.jsp?loginkey=zoomStart&mapId=wmsk_webatlasberlin@senstadt

> Nutzungsbedingungen: 
Nutzungsbedingungen: Für die Verwendung der Daten gelten folgende Nutzungsbestimmungen: http://www.stadtentwicklung.berlin.de/geoinformation/download/nutzIII.pdf - Der Quellenvermerk gemäß §2 lautet "Geoportal Berlin / [Titel des Datensatzes]". 

With the specification for OSM, that a changeset source is valid to satisfy this license, see https://forum.openstreetmap.org/viewtopic.php?pid=367583#p367583.

### TODO: What does `"attribution"."required": true` do?
I think we discussed this before. All Berlin layer don't have this required flag. Is that correct?

## Is the license compatible with OSM?
YES

## Attribution (text, URL, required?)
Geoportal Berlin / WebAtlas Berlin

## Privacy policy (URL)
./.

## Icon
./.

## Bounding polygon
See PR.

**Comment by tordans** [comment]
@bhousel as a follow up to https://github.com/osmlab/editor-layer-index/issues/538

The license here requires the `attribution.text` to be written in the changeset.
Does this only happen if `attribution.required` is `true`?

Or does the `attribution.required` only specify, that the `attribution.text` text is visible on the map/to the mapper? Or something else…

Does iD even change its behavior based on attribution.required=true/false? Do you know how other editors handle it?

Thanks for any clarification.

**Comment by bhousel** [comment]
> The license here requires the attribution.text to be written in the changeset.
Does this only happen if attribution.required is true?

Sorry @tordans, this is the first time I have heard of a requirement like this. 

iD doesn’t change its behavior, but we do always show the attribution on screen while the user is editing.

**Comment by grischard** [comment]
@tordans would it be acceptable to have the source shown in the changeset source tags?

**Comment by tordans** [comment]
Hi @grischard 

**License**

I continued talking about the topic of the license in other channels, including https://github.com/openstreetmap/iD/issues/6300#issuecomment-495860839. Sorry for not updating it here. For now, IMO this topic is postponed. Mainly cause the requirements are not clear and any action would be something like premature optimization at this point. I am waiting for someone in the german Community who knows more about this, like DD1GJ (I asked him via email a while back).

~~However, I will make a PR to make the Berlin-Data "attribution.required: true" if they are not, yet.~~
_Update: I did this a while back, they are all required already._

Another par of this is: I think the documentation of the layer index could more clearly state that this flag is used only when displaying the data-layer, not as an attribution of changesets (in the editors I know of). Is there a best place in this repo where you would add such an explanation? 

**This Layer**

`jochenklar` started experimenting with a proxy that would make it a lot easier to use the data as a TMS layer. This WIP thing is discussed at https://github.com/jochenklar/berlin-maps/issues/4#issuecomment-509356086. ATM the layer is IMO too slow and the server fails from time to time so I think we should wait to add it to the layer indes.
